src/phoenix/session/client.py

```diff
@@ -265,7 +265,37 @@ def log_traces(self, trace_dataset: TraceDataset, project_name: Optional[str] =
                 },
             ).raise_for_status()
 
-    def upload_dataset_table(
+    def download_dataset_examples(
+        self,
+        dataset_id: str,
```

Comment:
Should we allow users to download by name rather than ID to make it more human-friendly?

Reply:
We could since name is unique. But not sure if non-ascii characters could be an issue.